flake.nix

```diff
@@ -0,0 +1,37 @@
+{
+  inputs.golem-examples.url = "github:golemcloud/golem-examples";
```

Comment:
Why is `golem-examples` special for nix, and not our other dependencies? (For example `golem-wasm-rpc`, `golem-wasm-ast`, `golem-wit`, etc)

Reply:
I had issue building golem-examples when building golem-cli from nix
I think it was more "side effecty" setup.rs then the other projects, it was missing "Inflector" package. I was able to solve it referencing it on the nix build of golem-cli.
update-deps in the dev shell (commands.nix) will update it to latest version